src/Common/REST/Generated/code-model-v1

```diff
@@ -0,0 +1,81 @@
+{
+ "$id": "1",
+ "name": "AzureSignalRServiceRESTAPI",
+ "namespace": "Microsoft.Azure.SignalR",
+ "modelsName": "Models",
+ "apiVersion": "v1",
+ "baseUrl": "http://localhost",
+ "operations": [
+  {
+   "$id": "2",
+   "methods": [
+    {
+     "$id": "3",
+     "name": {
+      "$id": "4",
+      "fixed": false,
+      "raw": "GetHealthStatus"
+     },
+     "group": {
+      "$id": "5",
+      "fixed": false,
+      "raw": "HealthApi"
+     },
+     "serializedName": "HealthApi_GetHealthStatus",
+     "url": "/api/v1/health",
+     "isAbsoluteUrl": false,
+     "httpMethod": "head",
+     "responses": {
+      "OK": {
+       "$id": "6",
+       "extensions": {
+        "$id": "7",
+        "x-ms-metadata": {
+         "apiVersions": [
+          "v1"
+         ],
+         "filename": [
+          "mem:///109?oai3.shaken.json"
+         ],
+         "name": "paths·a2exod·api-v1-health·head·responses·200",
+         "originalLocations": [
+          "file:///C:/Users/zityang/source/repos/azure-signalr/src/Common/REST/health.json#/components/responses/paths·a2exod·api-v1-health·head·responses·200"
```

Comment:
Does it matters? I can see your alias

Reply:
Can it be relative path?

Reply:
I haven't found settings about this file, and I find that there are no these file in azure-sdk-for-net repo, so I just deleted it.